Core/src/main/java/com/plotsquared/core/plot/expiration/ExpireManager.java

```diff
@@ -60,6 +60,10 @@
 
 public class ExpireManager {
 
+    /**
+     * @deprecated Use {@link PlotPlatform#expireManager()} instead
```

Comment:
Build is failing here `error: reference not found`. Ensure it's imported or is referenced by FQN